ldap/src/main/java/org/springframework/security/ldap/server/UnboundIdContainer.java

```diff
@@ -108,23 +116,34 @@ public void start() {
 			this.running = true;
 		} catch (LDAPException ex) {
 			throw new RuntimeException("Server startup failed", ex);
+		} catch (IllegalArgumentException ex){
+			throw ex;
 		}
 
 	}
 
 	private void importLdif(InMemoryDirectoryServer directoryServer) {
-		if (StringUtils.hasText(this.ldif)) {
-			try {
-				Resource[] resources = this.context.getResources(this.ldif);
-				if (resources.length > 0 && resources[0].exists()) {
+		checkFilePath(this.ldif);
+		try {
+
+			Resource[] resources = this.context.getResources(this.ldif);
+			if (resources.length > 0
+					&& resources[0].exists()
+			) {
```

Comment:
There is still no exception thrown when the file does not exist.
We need to check if the LDIF exists and throw an exception if it does not.

```suggestion
			if (resources.length > 0) {
				if (!resources[0].exists()) {
					throw new IllegalArgumentException("Could not find LDIF " + this.ldif);
				}
```